### PR TITLE
feat: add component stress testing command

### LINKED ISSUE #288: Move component stress tests out of the main form definition & add flag to start a stress tested instance
__What should we do?__

Currently we have a test file with 10000 value entries for the Taglist. This should ideally not interrupt regular use cases, and we should stress test more components than just the Taglist. Hence I suggest we should move this into their own form definition and stress test all components that should work with larger datasets.

See: https://github.com/bpmn-io/form-js/blob/739e52f9093a08f68244ec9060f92123e0879397/packages/form-js-viewer/test/spec/form.json#L370

Also now that dynamic data works, we should use dynamic data instead of massive form definitions to make this happen, avoids us having 10000 lines of values in our form definitions.

__Why should we do it?__

This would let us run stress tests on our components easily as a developer by simply running `npm run start:viewer -- --stressTest`, but keeping it optional. It would also make the codebase more succinct.

### PR DESCRIPTION
**Pull request by Skaiir**
Closes #288

### CODE REVIEW

**Review by nikku — COMMENTED** [review]
I like this change a lot!

image

File: package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "start": "npm run start:pkg -- @bpmn-io/form-js",
     "start:viewer": "npm run start:pkg -- @bpmn-io/form-js-viewer",
     "start:editor": "npm run start:pkg -- @bpmn-io/form-js-editor",
-    "start:playground": "npm run start:pkg -- @bpmn-io/form-js-playground"
+    "start:playground": "npm run start:pkg -- @bpmn-io/form-js-playground",
+    "stress": "cross-env SINGLE_START=stress npm run dev -- @bpmn-io/form-js"
```
Comment (nikku) [inline package.json:31]: Should we rename this to `start:viewer:stress` or something?

**Review by nikku — CHANGES_REQUESTED** [review]
Let's rename the `stress` command to follow existing patterns.

**Comment by Skaiir** [comment]
@nikku Do you mean to start:viewer:stress? I think I did that.

**Comment by nikku** [comment]
Perfect, did miss your change!

**Review by nikku — APPROVED** [review]

**Pull request merged**